Include/import.h

```diff
@@ -81,8 +81,6 @@ PyAPI_FUNC(int) PyImport_ImportFrozenModule(
     const char *name            /* UTF-8 encoded string */
     );
 
-PyAPI_DATA(PyTypeObject) PyNullImporter_Type;
```

Comment:
It was implemented in 3.2. Removing the implementation in 3.3 was technically the breakage of the stable ABI. But is too late to fix this.